src/layouts/Settings.jsx

```diff
@@ -768,39 +478,62 @@ export default class Settings extends Component {
                       <FormFieldHorizontal
                         title={title}
                         id={footerSetting}
-                        value={otherFooterSettings[footerSetting]}
+                        value={currState.otherFooterSettings[footerSetting]}
                         key={`${footerSetting}-form`}
                         errorMessage={errors.otherFooterSettings[footerSetting]}
                         isRequired={false}
-                        onFieldChange={this.changeHandler}
+                        onFieldChange={changeHandler}
                       />
                     )
-                  })}
-                </div>
-                <br />
-                <br />
-              </div>
-              <div className={elementStyles.formSave}>
-                <LoadingButton
-                  label="Save"
-                  disabled={hasErrors}
-                  disabledStyle={elementStyles.formSaveButtonDisabled}
-                  className={
-                    hasErrors
-                      ? elementStyles.formSaveButtonDisabled
-                      : elementStyles.formSaveButtonActive
                   }
-                  callback={this.saveSettings}
-                />
+                )}
               </div>
+              <br />
+              <br />
+            </div>
+            <div className={elementStyles.formSave}>
+              <LoadingButton
```

Comment:
Just a small feature request, the designers/ Nat has previously requested to pin the Save button to the bottom of the Settings screen (since this is done for all the rest of our pages), would it be possible to implement this in this refactor?

Reply:
Thanks for the reminder - changed in https://github.com/isomerpages/isomercms-frontend/pull/666/commits/4b946ad083cbd1a47d2a9949ad7d1b70a4e483b3